src/System.Net.WebHeaderCollection/pkg/System.Net.WebHeaderCollection.pkgproj

```diff
@@ -3,7 +3,7 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
     <ProjectReference Include="..\ref\System.Net.WebHeaderCollection.csproj">
-      <SupportedFramework>net46;netcore50;netcoreapp1.0;$(AllXamarinFrameworks)</SupportedFramework>
```

Comment:
Since 'netcore50' is being dropped, does that mean that 'uap10.1' will need to be added back explicitly some time later in order to get UWP support?